packages/client/tests/functional/_utils/setupTestSuiteMatrix.ts

```diff
@@ -134,10 +134,21 @@ function setupTestSuiteMatrix(
 
       afterAll(async () => {
```

Comment:
Note: this `afterAll` hook here runs for every functional Client test, and actually disables opportunities for other, most specific `afterAll` hooks, as this is the first one that runs, and causes the global `prisma` variable to turn to `undefined` before the next `afterAll` hooks are hit.